### DISCUSSION TITLE
Unstable Features

### DISCUSSION THREAD

**Discussion by coolcatcoder** [bevyengine/bevy#16323] [Ideas]
Why not imitate rust unstable features in bevy? When there are multiple solutions to a problem, then just put the solutions behind features with unstable_ in front of them, which would allow users of the bevy main branch to try them out and give feedback.
(I am just a simple bevy user though, so I have no idea how this would affect whatever is going on with how code is structured in bevy. If this suggestion is absolutely atrocious, then I apologise for my ignorance.)

**Comment by mockersf** [comment]
If you think it's a good idea, you can comment on this RFC: https://github.com/bevyengine/rfcs/pull/52
